include/internal/rand.h

```diff
@@ -10,8 +10,8 @@
 #ifndef HEADER_DRBG_RAND_H
 # define HEADER_DRBG_RAND_H
 
-/* In CTR mode, use derivation function ctr_df */
-#define RAND_DRBG_FLAG_CTR_USE_DF            0x2
+/* In CTR mode, disable derivation function ctr_df */
+#define RAND_DRBG_FLAG_CTR_NO_DF            0x1
```

Comment:
oops.  my fault?

Reply:
Looks like you changed the value of `RAND_DRBG_FLAG_CTR_USE_DF` inadvertently for `0x1` to `0x2`. Number `0x2` used to be the test flag.

```
/* DRBG external flags */
/* Flag for CTR mode only: use derivation function ctr_df */
#  define DRBG_FLAG_CTR_USE_DF            0x1
/* PRNG is in test state */
#  define DRBG_FLAG_TEST                  0x2
```

But it does not make a difference. However, the inversion of the meaning from opt-in to opt-out is new.


Apropos test flag: was it really a good idea to rip out the test code in view of FIPS 3.0? Maybe the selftest should be added back after the code freeze?